tools/istio-iptables/pkg/builder/iptables_builder_test.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package builder
+
+import (
+	"testing"
+)
+
+func Test_BuildV6Restore(t *testing.T) {
+	iptables := NewIptablesBuilder()
+	expected := ""
```

Comment:
You need more test cases for the `*Restore` methods. Just the empty set of rules is not sufficient.
Document this as a `TODO(abhide)` comment for now, since these methods haven't been implemented yet.

Reply:
yeah.. instead of ignoring I added empty ruleset test. This test will start failing once Restore methods are implemented.